app/src/test/java/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingScreenTest.kt

```diff
@@ -1477,6 +1483,186 @@ class VaultItemListingScreenTest : BaseComposeTest() {
+            VaultItemListingEvent.Fido2UserVerification(
+                isRequired = true,
+                selectedCipherView = selectedCipherView,
+            ),
+        )
+
+        verify {
+            viewModel.trySendAction(
+                VaultItemListingsAction.UserVerificationSuccess(selectedCipherView),
+            )
+        }
+    }
+
+    @Test
+    fun `promptForUserVerification onCancel should send UserVerificationCancelled action`() {
+        val selectedCipherView = createMockCipherView(number = 1)
+        every { biometricsManager.isUserVerificationSupported } returns true
+        every {
+            biometricsManager.promptUserVerification(
+                any(),
+                onCancel = captureLambda(),
+                any(),
+                any(),
+            )
+        } answers {
+            lambda<() -> Unit>().invoke()
+        }
+
+        mutableEventFlow.tryEmit(
+            VaultItemListingEvent.Fido2UserVerification(
+                isRequired = true,
+                selectedCipherView = selectedCipherView,
+            ),
+        )
+
+        verify {
+            viewModel.trySendAction(
+                VaultItemListingsAction.UserVerificationCancelled,
+            )
+        }
+    }
+
+    @Test
+    fun `promptForUserVerification onLockOut should send UserVerificationLockOut action`() {
+        val selectedCipherView = createMockCipherView(number = 1)
+        every { biometricsManager.isUserVerificationSupported } returns true
+        every {
+            biometricsManager.promptUserVerification(
+                any(),
+                any(),
+                onLockOut = captureLambda(),
+                any(),
+            )
+        } answers {
+            lambda<() -> Unit>().invoke()
+        }
+
+        mutableEventFlow.tryEmit(
+            VaultItemListingEvent.Fido2UserVerification(
+                isRequired = true,
+                selectedCipherView = selectedCipherView,
+            ),
+        )
+
+        verify {
+            viewModel.trySendAction(
+                VaultItemListingsAction.UserVerificationLockOut,
+            )
+        }
+    }
+
+    @Test
+    fun `promptForUserVerification onError should send UserVerificationFail action`() {
+        val selectedCipherView = createMockCipherView(number = 1)
+        every { biometricsManager.isUserVerificationSupported } returns true
+        every {
+            biometricsManager.promptUserVerification(
+                any(),
+                any(),
+                any(),
+                onError = captureLambda(),
+            )
+        } answers {
+            lambda<() -> Unit>().invoke()
+        }
+
+        mutableEventFlow.tryEmit(
+            VaultItemListingEvent.Fido2UserVerification(
+                isRequired = true,
+                selectedCipherView = selectedCipherView,
+            ),
+        )
+
+        verify {
+            viewModel.trySendAction(
```

Comment:
Do we want to verify all the methods that were mocked? In particular,`biometricsManager.promptUserVerification` here?

Reply:
We can infer that `promptUserVerification` is called because the action is being sent by capturing the lambda and invoking it. If `promptUserVerification` is never called, `onError` is never invoked and `UserVerificationFail` is not sent so the test would fail.